server/base/src/main/java/org/apache/accumulo/server/util/PropUtil.java

```diff
@@ -65,6 +65,19 @@ protected static void validateProperties(final ServerContext context,
         }
         throw new IllegalArgumentException(exceptionMessage + propStoreKey + " name: "
             + prop.getKey() + ", value: " + prop.getValue());
+      } else if (prop.getKey().equals(Property.TABLE_CLASSLOADER_CONTEXT.getKey())) {
+        ClassLoader cl;
+        try {
+          // If the context cannot be resolved, then this call should throw an error
+          cl = ClassLoaderUtil.getClassLoader(prop.getValue());
+        } catch (RuntimeException re) {
```

Comment:
`ClassLoaderUtil.getClassLoader(contextName)` calls `ContextClassLoaderFactory.getClassLoader(contextName)`. The javadoc there says to throw a RuntimeException - see [here](https://github.com/apache/accumulo/blob/2.1/core/src/main/java/org/apache/accumulo/core/spi/common/ContextClassLoaderFactory.java#L60). If I'm able to add the method you suggested above, then this might be moot.